### PR TITLE
Fixing typo "refere here" -> "refer here"

### PR DESCRIPTION
**Pull request by LoicPoullain**
Thank you for submitting a Pull Request to the Cheat Sheet Series. 

Please make sure that for your contribution:

- [ ] In case of a new Cheat Sheet, you have used the [Cheat Sheet template](https://github.com/OWASP/CheatSheetSeries/blob/master/templates/New_CheatSheet.md).
- [ ] All the markdown files do not raise any validation policy violation, see policy [here](https://github.com/OWASP/CheatSheetSeries#editor--validation-policy).
- [ ] All the markdown files follow these [format rules](https://github.com/OWASP/CheatSheetSeries#conversion-rules).
- [ ] All your assets are stored in the **assets** folder.
- [ ] All the images used are in the **PNG** format.
- [ ] Any references to website have been formatted as [TEXT](URL)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: defensive code proposed is really an effective remediation? Please verify it works!).
- [ ] The CI build of your PR pass, see the build status [here](https://travis-ci.org/OWASP/CheatSheetSeries/pull_requests).

If your PR is related to an issue. Please end your PR text with the following line:

```
This PR covers issue #<insert number here>.
```

Thank you again for your contribution :smiley:

### CODE REVIEW

File: cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ For these cases, attempting to retrofit this pattern in existing applications re
...
...to prevent replay attacks) using a unique key available only on the server (AES256-with GCM mode/GCM-SIV is recommended. Usage of ECB mode is strictly not recommended. If you would like to use any other block cipher mode of operation, refer [here](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Cryptographic_Storage_Cheat_Sheet.md) and [here](https://en.wikipedia.org/wiki/Block_cipher_mode_of_operation) for more information). This token is returned to the client and embedded in a hidden field for forms, in the request-header/parameter for AJAX requests. On receipt of this request, the server reads and decrypts the token value with the same key used to create the token. Inability to correctly decrypt suggest an intrusion attempt (recommend to block and log the attack for incident response purposes). Once decrypted, the users sessionId and timestamp contained within the token are validated; session-id is compared against the currently logged in user, and the timestamp is compared against the current time to verify that its not beyond the defined token expiry time. If session-id matches and the timestamp is under the defined token expiry time, request can be allowed.
```
Comment (righettod) [inline cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Can you change every links to others CS to relative one ?
As all CS in the same folder so `https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/` part can be removed.

Comment (LoicPoullain) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Done!

Comment (righettod) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Thanks.

The part `./` is not needed too 😃 

You can just put the name of the CS like this 

`refer [here](Cryptographic_Storage_Cheat_Sheet.md)`

Comment (LoicPoullain) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Links updated

Comment (righettod) [reply cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md]: Thanks you very much 😃

**Pull request merged**